### PR TITLE
Explore then commit module

### PR DESCRIPTION
**Pull request by aidanclaffey**
Implemented the explore-then-commit policy in `obp_extensions/policy.py`. Additionally, added a notebook that walks through the policy simulation of for both Explore-Then-Commit and Epsilon Greedy, and does OPE estimation as well.

### CODE REVIEW

**Review by ChrisIck — APPROVED** [review]
LGTM

**Review by ChrisIck — COMMENTED** [review]
Looks good to me, I didn't get a chance to pull the notebook and play around but I'll naively assume it's all good. Won't merge until I take a look tomorrow.

File: sd_bandits/obp_extensions/policy.py [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+        Number of samples used in a batch parameter update.
+    
+    min_n: int, default=1
+        Minimum number of times to explore each action before commiting
+        to one
+
+    random_state: int, default=None
+        Controls the random seed in sampling actions.
+
+    policy_name: str, default=f'egreedy_{epsilon}'.
```
Comment (daturkel) [inline sd_bandits/obp_extensions/policy.py]: should be `default=f'etc{min_n}'`

**Comment by daturkel** [comment]
Made one small comment on a typo in the docstring but otherwise looks great

**Review by daturkel — APPROVED** [review]

**Pull request merged**